plugins/cratedb/database_credentials.go

```diff
@@ -0,0 +1,82 @@
+package cratedb
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.DatabaseCredentials,
+		DocsURL:       sdk.URL("https://crate.io/docs/crate/crash/en/latest/run.html#environment-variables"),
+		ManagementURL: sdk.URL("https://console.cratedb.cloud/account/settings"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Host,
+				MarkdownDescription: "CrateDB host to connect to.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.Port,
+				MarkdownDescription: "Port used to connect to CrateDB.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.User,
+				MarkdownDescription: "CrateDB user to authenticate as.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.Password,
+				MarkdownDescription: "Password used to authenticate to CrateDB.",
+				Secret:              true,
+			},
+			{
+				Name:                fieldname.Database,
+				MarkdownDescription: "Database name to connect to.",
+				Optional:            true,
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryCrateDBConfigFile(),
+		)}
+}
+
+var defaultEnvVarMapping = map[string]sdk.FieldName{
+	"CRATEPW": fieldname.Password, // TODO: Check if this is correct
+}
+
+// TODO: Check if the platform stores the Database Credentials in a local config file, and if so,
+// implement the function below to add support for importing it.
+func TryCrateDBConfigFile() sdk.Importer {
```

Comment:
For the file importer we could use the [information provided here](https://crate.io/docs/crate/crash/en/latest/run.html#user-configuration-directory) to import any valuable settings from the config file. I'm not familiar with `crash`'s config file but we are interested to import any of the host/port/user/password fields if those are present there. If they are not present, feel free to remove the file importer, and rely solely on the env var importer that you already have in place

Reply:
I have checked config file but there is not found information which can be said cred

Reply:
👍